src/main/java/edu/itu/cavabunga/controller/wrapper/ComponentPropertyResponse.java

```diff
@@ -0,0 +1,42 @@
+package edu.itu.cavabunga.controller.wrapper;
+
+import edu.itu.cavabunga.core.entity.ComponentProperty;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class ComponentPropertyResponse {
```

Comment:
All the response class almost same, they should inherit from a base response class

Reply:
The problem about this thing is little different. Kind a bug is exists in Jackson library(people also complainig about it on github) to force me to do this. For example, if you wanna call a calendar object which has event sub object. Jackson will create a "@type: XXXX" column in the json response for all nested and parent object. Library also expects that column when converts http request json data to objects. Basically that "@type: XXX" thing is needed to determine which class you have. When you add object into response class and send as a http response. Parent object's "@type: XXX" json column dissapears(only the parent's @type column). So you should but objects in the type of your object's type array(or list or whatever)  to appear correctly in the http response. Its little bit tricky but i'll consider your review and will correct that